install_config/install/disconnected_install.adoc

```diff
@@ -193,7 +193,7 @@ will not recycle, potentially causing errors.
 . Pull all of the required {product-title} containerized components for the
 additional centralized log aggregation and metrics aggregation components.
 ifdef::openshift-enterprise[]
-Replace `<tag>` with `{latest-int-tag}` for the latest version.
+Replace `<tag>` with `3.4` for the latest version.
```

Comment:
You can leave the {latest-int-tag} here, but update the value for latest-int-tag: 3.4.1 value at the top of the file, in the file's meta data. I'm looking at the comments in the BZ and it looks like the proposed new value should be v3.4.